hyperlane/scripts/docker-entrypoint.sh

```diff
@@ -34,6 +34,8 @@ if [[ ! -f "$CONFIG_FILE" ]]; then
 
   echo "Configuring remote router for warp route on cosmosnative..."
   hyp enroll-remote-router celestia-validator:9090 0x726f757465725f61707000000000000000000000000000010000000000000000 1234 0x000000000000000000000000345a583028762De4d733852c9D4f419077093A48
+
+  hyp setup-zkism celestia-validator:9090 reth:8545 ev-node-evm-single:7331
```

Comment:
and then this will deploy the zk ism and overwrite the ids on the mailbox and token